episodes/20-reflecting.md

```diff
@@ -46,24 +46,41 @@ the trial run are used to improve the quality of your lesson.
+- What did not work as well?
+- How close were your time estimates to the actual time needed to teach the material and finish exercises?
+- How did the audience perceive the difficulty level of the material?
 - What will you do differently next time?
-- What will you change in your material you taught?
-  
+- What will you change in the material you taught?
+- What will you change in the way you collect feedback in future pilots?  
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+Moving forward from your first pilot, the next steps to consider are:
 
+- Recording all the identified changes and improvements that can be made to the lesson material and assigning a person from the team responsible for each task. The above exercise provided some reflection points after a trial run; in terms of lesson content they may translate into the following considerations:
+  - is the material too dense, does it need extra explanations?
+  - would adding a diagram help explain or curate things better?
+  - is there too much content for one episode and do you need to split into smaller teaching units?
+  - do you need to re-organise and move some content around to improve the flow and narrative?
+  - are there enough exercises and practical work? 
+  - do you need to realign your lesson objectives and key messages?
+- Deciding if some of the feedback you will not action. Remember, you do not need to respond to every piece of feedback you receive. For example, it is easy to fall down the rabbit hole of adding extra material just because someone suggested it may be a useful thing - sometimes adding a link to extra reading is sufficient. You need to be able to draw a line under any extra modification suggestions to keep you in scope and on schedule. 
+- Schedule follow-up co-working sessions with your team to carry on working on fixing issues and adding new content to maintain the momentum.
+- Think about the timeline for your next pilot(s), even provisionally, to help you set milestones and targets to  work towards.
+- Consider writing up your pilot experience in a mini blog post that you can share with the community.
+
+Designing and developing quality lessons is hard - there are many pieces of a puzzle that have to come into place: both pedagogical and organisational. Do not be disheartened with the amount and type of feedback you may have received. Even Carpentries lessons that have been around for 10+ years receive improvement suggestions and fixes almost on a daily basis. From our experience, bigger lessons that are delivered over a few days require several full pilots before they can even be considered for a [beta](https://carpentries.github.io/curriculum-development/the-lesson-life-cycle.html) release. Planning smaller lesson trials (where you test only a portion of a lesson) and doing them more often with a friendly audience from your local groups and close colleagues is more manageable and will help you make steady progress.
```

Comment:
I would like to tie this back to the lesson design process outlined in the Target Audience chapter - we can point to the flowchart (when it exists! should be something like figure 2 in https://f1000research.com/documents/9-1377) and identify Evaluation as one of the key steps in the process. I think this will be a good way of reminding the participants about what they learned in the previous sessions, and hopefully give them some satisfaction that they have now completed one full cycle of the process, at least for one chapter of their lessons! No need for that to prevent this from being merged - I'll open an issue to remind us to add this in.

Reply:
The relevant issue opened by @tobyhodges: https://github.com/carpentries/lesson-development-training/issues/40